pymatgen/io/vasp/outputs.py

```diff
@@ -3628,28 +3703,32 @@ def from_file(cls, filename: str) -> Self:
         return cls(poscar, data, data_aug=data_aug)
 
     @property
-    def net_magnetization(self):
-        """Net magnetization from Chgcar"""
+    def net_magnetization(self) -> float | None:
+        """Net magnetization from Chgcar."""
+        # TODO: (DanielYang59): default to 0.0 for easier typing?
         return np.sum(self.data["diff"]) if self.is_spin_polarized else None
```

Comment:
Should we use `0.0` as default magnetic moment for easier typing?

Reply:
Make sense to me. I'm not quite experienced with non-collinear calculations so let's leave it as is :)